### PR TITLE
Matiasjj/deployed version

### PR DESCRIPTION
**Pull request by MatiasJJ**
Here in the deployment version there are some changes that we made during the setup. I would like to have those in the template as well. Also I added 
```
- name: CMD_ALLOW_FREEURL
              value: 'true'
```
which I think is relevant also.

### CODE REVIEW

File: hedgedoc-deployment.yml [hunk 1]
```diff
@@ -0,0 +1,102 @@
+apiVersion: apps/v1
```
Comment (lvarin) [inline hedgedoc-deployment.yml]: This file should not be here

Comment (MatiasJJ) [inline hedgedoc-deployment.yml]:
```suggestion

```

File: hedgedoc-deployment.yml [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+    - lastTransitionTime: '2022-11-02T13:42:50Z'
+      lastUpdateTime: '2022-11-03T12:21:25Z'
+      message: ReplicaSet "app-66678d56f7" has successfully progressed.
+      reason: NewReplicaSetAvailable
+      status: 'True'
+      type: Progressing
+  observedGeneration: 3
+  readyReplicas: 1
+  replicas: 1
+  updatedReplicas: 1
```
Comment (lvarin) [inline hedgedoc-deployment.yml]: I meant the whole file

Comment (lvarin) [reply hedgedoc-deployment.yml]: This whole file must be removed

**Review by lvarin — CHANGES_REQUESTED** [review]
If the file is removed, I will approve the PR

File: hedgedoc-deployment.yml (hunk 2 above)
Comment (MatiasJJ) [reply hedgedoc-deployment.yml]: Yes now I see

**Review by lvarin — APPROVED** [review]
LGTM

**Pull request merged**